backend/config.js

```diff
@@ -0,0 +1,19 @@
+const userId = 3376787;
```

Comment:
update .gitignore so this file isn't committed

Reply:
+1